src/api.ts

```diff
@@ -327,16 +334,38 @@ function createApi({
         'INSERT INTO Tileset (id, tilejson, format, upstreamTileUrls) ' +
           'VALUES (:id, :tilejson, :format, :upstreamTileUrls)'
       ).run({
-        id,
+        id: tilesetId,
         format: tilejson.format,
         tilejson: JSON.stringify(tilejson),
         upstreamTileUrls: JSON.stringify(tilejson.tiles),
       })
 
       const result = {
         ...tilejson,
-        tiles: [getTileUrl(id)],
-        id,
+        id: tilesetId,
+        tiles: [getTileUrl(tilesetId)],
+      }
+
+      // Create raster style if tileset format is a raster format
+      if (tilejson.format !== 'pbf') {
+        const rasterStyle = createRasterStyle({
+          // TODO: Come up with a better default name
+          name: tilejson.name || `Style ${tilesetId.slice(-4)}`,
```

Comment:
Just highlighting this if we ever look back on it or if anyone's got better ideas